### PR TITLE
Add test and import testify.assert tool

### PR DESCRIPTION
**Pull request by jyz0309**
Add some test for tools, and import testify.assert, just for the test more convenient : )

### CODE REVIEW

File: go.sum [hunk 1]
```diff
@@ -15,9 +15,10 @@ github.com/pmezard/go-difflib v1.0.0/go.mod h1:iKH77koFhYxTK1pcRnkKkqfTogsbg7gZN
 github.com/stretchr/objx v0.1.0/go.mod h1:HFkY916IF+rwdDfMAkV7OtwuqBVzrE8GR6GFx+wExME=
 github.com/stretchr/testify v1.6.1 h1:hDPOHmpOpP40lSULcqw7IrRb/u7w6RpDC9399XyoNd0=
 github.com/stretchr/testify v1.6.1/go.mod h1:6Fq8oRcR53rry900zMqJjRRixrwX3KX962/h/Wwjteg=
+github.com/stretchr/testify v1.7.0 h1:nwc3DEeHmmLAfoZucVR881uASk0Mfjw8xYJ99tb5CcY=
+github.com/stretchr/testify v1.7.0/go.mod h1:6Fq8oRcR53rry900zMqJjRRixrwX3KX962/h/Wwjteg=
```
Comment (yixinglu) [inline go.sum:19]: I find that testify v1.6.1 had been imported, could you change its version from v1.7.0 to v1.6.1 ?

**Comment by jyz0309** [comment]
cc @yixinglu

File: pkg/csv/csv_test.go [hunk 2]
```diff
@@ -23,3 +23,7 @@ func TestCsvWriter(t *testing.T) {
 		t.Fatal(w.Error())
 	}
 }
+
+func TestCsvReader(t *testing.T) {
+
```
Comment (yixinglu) [inline pkg/csv/csv_test.go:28]: Do you want to fill something in this test function later ?

**Review by yixinglu — COMMENTED** [review]
Thanks for your contributions

File: pkg/csv/csv_test.go (hunk 2 above)
Comment (jyz0309) [reply pkg/csv/csv_test.go]: sry, have been removed it :) maybe I will add it in the next pr

File: go.sum (hunk 1 above)
Comment (jyz0309) [reply go.sum]: done

**Review by yixinglu — APPROVED** [review]
LGTM

**Pull request merged**